pkg/resource/node/scanner.go

```diff
@@ -34,7 +35,7 @@ func (s *Scanner) ID() resource.ScannerID {
 }
 
 func (s *Scanner) Scan(ctx context.Context, runtime *runtime.Info) (resource.ScanConverter, error) {
-	ctx, span := otel.Tracer("").Start(ctx, "kmc.node_scan")
+	ctx, span := otel.Tracer("").Start(ctx, "kmc.node_scan", kmcotel.SpanAttributes(runtime))
```

Comment:
imho "kmc" is the service name which is set using env-variables

Reply:
the `kmc` prefix is now removed from the span names